integration-tests/tests/server.ts

```diff
@@ -86,7 +80,7 @@ class CaptureServer extends EventEmitter {
   constructor() {
     super();
     this.app = express();
-    this.app.get('/h', this.onGetRequest.bind(this));
+    this.app.get('*', this.onGetRequest.bind(this));
```

Comment:
Are there other get endpoints besides `/h`?

Reply:
It looks like maybe `/api/add_user_properties` and `/api/identify_v3` could also be get requests. Is that correct?

Reply:
That's right.  Since all GET requests can be represented as a `URL` object, I just stash them in the same place and have a single method that gets each type: `expectPixelRequest`.  A good example of how this is used is `expectIdentify(identity: string)`:

https://github.com/heap/react-native-heap/blob/26370e65e6cdc389271b2e05ceb96fc1be733a0a/integration-tests/tests/util/server.ts#L347-L364